### PR TITLE
[feature](pipelineX) add mem control in local exchange sink

### PR DESCRIPTION
**Pull request by Mryange**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run p0

**Comment by Mryange** [comment]
run p0 10

**Comment by Mryange** [comment]
run p0

**Comment by Mryange** [comment]
run pipelinex_p0

**Comment by Mryange** [comment]
run pipelinex_p0

**Comment by Mryange** [comment]
run p0

**Review by Gabriel39 — APPROVED** [review]

**Review by Gabriel39 — CHANGES_REQUESTED** [review]
P0 failed

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run pipelinex_p0

**Comment by Mryange** [comment]
run buildall

File: be/src/vec/sink/vdata_stream_sender.h [hunk 1]
```diff
@@ -342,6 +342,11 @@ class Channel {
 
     void set_receiver_eof(Status st) { _receiver_status = st; }
 
+    auto local_recvr() {
```
Comment (yiguolei) [inline be/src/vec/sink/vdata_stream_sender.h:345]: 返回值别auto了

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run p0

**Comment by Mryange** [comment]
run p0

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run p0

**Pull request closed without merging**